### PR TITLE
Demo schedule

### PR DESCRIPTION
**Pull request by Jeremy-hao**
Tomorrow I will fill up the XML page. Please give me some images about what kinds of XML page we want. Currently, I have no idea about what this page should look like.

### CODE REVIEW

**Comment by nout-kleef** [comment]
I think the following is a nice example for an overview (left) and time selection (right):
preview

The following image is another example of how the overview of a few schedules might look:
6508b9a6b4b1a20c7799e9e06e83703d


Of course, we will need to include whether we want the blind to either open or close at that time, so these are not perfect examples.

**Comment by adamb56789** [comment]
Instead or as well as open or close you could make it so you can choose which level you want to move to. Maybe you could use the same slider that is used in the control part ([layout file](https://github.com/SDP-group22/Helio-app/blob/main/app/src/main/res/layout/control_list_item.xml), [class](https://github.com/SDP-group22/Helio-app/blob/main/app/src/main/java/com/helio/app/ui/control/ControlRecViewAdapter.java)).

**Comment by Jeremy-hao** [comment]
The API of time picker didn't work. I wrote an annotation in the FragmentTimerActivity.

**Review by adamb56789 — CHANGES_REQUESTED** [review]
I think the instructions were a bit unclear. I made some boilerplate and template code to make the requirements for the scheduling pages more clear. Some additional recommendations, which are repeated in comments in the relevant files:
### The list items of the schedules recycler view
- [] The name if there is one
- [] Some sort of visual indicator which displays the target level of the motor
- [] An indicator of which days it is active
- [] A switch (https://material.io/components/switches) which controls whether this schedule is on or off
### The settings page for a schedule
- [] The time, which opens a time picker (https://material.io/components/time-pickers) when clicked on which displays the target level of the motor
- [] The days of the week, as 7 circles containing the letters MTWTFSS which when pressed become highlighted to control the days that it gets activated
- [] A text field (https://material.io/components/text-fields) to set the name
- [] A slider like the ones in control_list_item to set the target level
- [] Some sort of way to select which blinds this schedule applies to, such as using a drop down menu and checkboxes (https://material.io/components/checkboxes)

Please also try to make your commit messages more descriptive, also try to split up your commits into smaller chunks. For example "Added time picker for schedules".
Try to connect it to the backend if you can, I've left in some code copied from the blinds code that should get that started. You may have to make a couple of new methods in UserDataViewModel.

File: app/src/main/res/layout/schedule_list_item.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    -->
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/border"
+        android:paddingVertical="5dp">
+
+        <TextView
+            android:id="@+id/blindIcon"
```
Comment (adamb56789) [inline app/src/main/res/layout/schedule_list_item.xml:25]: I forgot to change the ID here, change it s=to something sensible.

**Comment by Jeremy-hao** [comment]
Currently, I still didn't finish the days of week UI and its function. I will finish it before today.

**Review by adamb56789 — COMMENTED** [review]
I'll take over finishing this ticket, can you work on the sensors?

**Comment by Jeremy-hao** [comment]
Yes, of course

**Pull request closed without merging**